python/paddle/tensor/linalg.py

```diff
@@ -3724,3 +3726,219 @@ def cdist(
     return paddle.linalg.norm(
         x[..., None, :] - y[..., None, :, :], p=p, axis=-1
     )
+
+
+def histogramdd(
+    sample, bins=10, range=None, density=False, weights=None, name=None
```

Comment:
according to [API naming conventions](https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/api_contributing_guides/api_design_guidelines_standard_cn.html), enter the name of Tensor using `x`, and the [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20231001_api_design_for_histogramdd.md) should also be modified synchronously

Reply:
OK, I will modify it soon~